runners/gearpump/pom.xml

```diff
@@ -70,26 +70,17 @@
                     <dependency>org.apache.beam:beam-sdks-java-core</dependency>
                   </dependenciesToScan>
                   <excludes>
-                    <!-- side input is not supported in Gearpump -->
+                    <!-- duplicated edges in Gearpump graph -->
                     <exclude>
-                      org.apache.beam.sdk.io.gcp.bigquery.BigQueryIOTest,
-                      org.apache.beam.sdk.io.CountingInputTest,
-                      org.apache.beam.sdk.io.CountingSourceTest,
-                      org.apache.beam.sdk.testing.PAssertTest,
-                      org.apache.beam.sdk.transforms.ApproximateUniqueTest,
-                      org.apache.beam.sdk.transforms.CombineTest,
-                      org.apache.beam.sdk.transforms.CombineFnsTest,
-                      org.apache.beam.sdk.transforms.CountTest,
-                      org.apache.beam.sdk.transforms.FlattenTest,
-                      org.apache.beam.sdk.transforms.ParDoTest,
-                      org.apache.beam.sdk.transforms.SampleTest,
-                      org.apache.beam.sdk.transforms.ViewTest,
-                      org.apache.beam.sdk.transforms.join.CoGroupByKeyTest
+                      org.apache.beam.sdk.transforms.FlattenTest
                     </exclude>
-                    <!-- merging windows is not supported in Gearpump -->
+                    <!-- assertion should have failed-->
                     <exclude>
-                      org.apache.beam.sdk.transforms.windowing.WindowingTest,
-                      org.apache.beam.sdk.util.ReshuffleTest
+                      org.apache.beam.sdk.testing.PAssertTest
+                    </exclude>
+                    <!-- only one side input is supported -->
```

Comment:
Most of the logic seems to support multiple side inputs - using a union value and writing to a `SideInputHandler`. If really only one side input is supported, I think we need to use a new JUnit `@Category` to exclude only the tests that use more than one side input. If we exclude `ParDoTest` we lose too much core functionality, including a lot of side input testing.